docs/style_guide.adoc

```diff
@@ -0,0 +1,152 @@
+
+.Good Example :smile:
+----
+= Top Header
+== Section 1
+== Section 2
+=== Section 2.1
+==== Section 2.1.1
+=== Section 2.2
+== Section 3
+----
+
+.Bad Example :cry:
+----
+= Top Header
+=== Section 1
+=== Section 2
+=== Section 2.1
+===== Section 2.1.1
+== Section 2.2
+= Section 3
+----
+
+=== Links
+
+Links should follow the following format:
+----
+link:/path/to/file{outfilesuffix}[Link Name]
+----
+The `{outfilesuffix}` parameter should replace the file type and is there to handle the translation from '.adoc' to '.html' when we publish this content to our web front end.
+
+=== Interlinking Content
+
+Whenever possible we try to have content refer to other guides & articles. Tying content together in a logical way helps us build up complete stories and solutions and even lead to other solutions.
+
+When linking to other content within the site, we like to take steps to make it as obvious as possible without ruining the flow of the doc. Here are some guidelines for formatting your links:
+
+1. Include the title of the guide in the link text. For example...
++
+----
+link:/guides/openshift-ha-installation{outfilesuffix}[Installing a Highly Available OpenShift Cluster]
+----
++
+rather than
++
+----
+link:/guides/openshift-ha-installation{outfilesuffix}{outfilesuffix}[click here!]
+----
++
+2. Provide some context as to why you are providing this link. For example...
++
+----
+The link:/articles/openshift-load-balancing{outfilesuffix}[External Load Balancing for OpenShift] provides an introduction to the strategies that can be employed within OpenShift.
+----
++
+3. Use a NOTE callout when there's no natural flow.
++
+----
+NOTE: The link:/articles/load_balancing{outfilesuffix}[External Load Balancing for OpenShift] provides an introduction to the strategies that can be employed within OpenShift.
+----
+
+=== Include a "What's Next?" section to encourage further reading
+
+We like to conclude our documents with a "call to action" or something of that sort. Whenever applicable, think about ending your doc with a "What's Next?" section with links out to further suggested playbooks guides, or other documents when a playbook guide doesn't yet exist (if you think it SHOULD exist, please link:https://github.com/rhtconsulting/openshift-playbooks/issues/new[Open an Issue]).
+
+== Content Principles
+
+All written content should follow these principles:
+
+* **Approachability:** Don't assume reader has prior knowledge
+* **Brevity:** Keep it simple, link to outside content for deeper dives
+* **Curation:** Amplify community best practices vs. any individual's point of view
+
+Content should maintain a light-hearted, but wise (think classy, not overly excited) tone. Open source is fun! Readers should feel inspired, not discouraged, by the tone of your writing, and they should trust you to help them get started.
+
+== Mentions
+
+When referring to people that use GitHub, use @mentions of their username instead of their full name.
```

Comment:
Again, this section is based on the [style guide for opensource.guide](https://github.com/github/opensource.guide/blob/master/docs/styleguide.md).

Where uncontained.io isn't hosted on GitHub, I'd actually prefer to use people's real names and link to their GitHub profiles, rather than using only their usernames. Or, if we keep this section, let's cite the source.